nox/sessions.py

```diff
@@ -208,12 +208,64 @@ def _run(self, *args, env=None, **kwargs):
             kwargs.setdefault("external", "error")
 
         # If we aren't using a virtualenv allow all external programs.
-        if not isinstance(self.virtualenv, VirtualEnv):
+        if not isinstance(self.virtualenv, (CondaEnv, VirtualEnv)):
+            kwargs["external"] = True
+
+        # Allow the global conda command from conda environments.
+        if isinstance(self.virtualenv, CondaEnv) and args[0] == "conda":
             kwargs["external"] = True
 
         # Run a shell command.
         return nox.command.run(args, env=env, path=self.bin, **kwargs)
 
+    def conda_install(self, *args, **kwargs):
+        """Install invokes `conda install`_ to install packages inside of the
+        session's environment.
+
+        To install packages directly::
+
+            session.conda_install('pandas')
+            session.conda_install('numpy', 'scipy')
+            session.conda_install('--channel=conda-forge', 'dask==2.1.0')
+
+        To install packages from a `requirements.txt` file::
+
+            session.conda_install('--file', 'requirements.txt')
+            session.conda_install('--file', 'requirements-dev.txt')
+
+        To install the current package without clobbering conda-installed
+        dependencies::
+
+            session.install('.', '--no-deps')
+            # Install in editable mode.
+            session.install('-e', '.', '--no-deps')
+
+        Additional keyword args are the same as for :meth:`run`.
+
+        .. _conda install:
+        """
+        if not isinstance(self.virtualenv, CondaEnv):
+            raise ValueError(
+                "A session without a conda environment can not install dependencies from conda."
+            )
+        if not args:
+            raise ValueError("At least one argument required to install().")
+
+        if "silent" not in kwargs:
+            kwargs["silent"] = True
+
+        self._run(
+            "conda",
+            "install",
+            "--yes",
+            "--use-index-cache",
```

Comment:
What if people don't want to use the index cache?

Reply:
Oops. You're right, removed. I added that so that I could code on the airplane.

Now they can add `--use-index-cache` from the `conda_install` function call, and by default it uses the default CLI params.